src/blocks/LinkPreview/Component.tsx

```diff
@@ -0,0 +1,49 @@
+import { CMSLink } from '@/components/Link'
+import { Media } from '@/components/Media'
+
+import type { LinkPreviewBlock as LinkPreviewBlockProps } from '@/payload-types'
+import { cn } from '@/utilities/cn'
+
+export const LinkPreviewBlock = (props: LinkPreviewBlockProps) => {
+  const { cards } = props
+
+  const numOfCols = cards?.length ?? 2
+
+  const colsClasses: { [key: number]: string } = {
+    2: 'lg:col-span-6',
+    3: 'lg:col-span-4',
+  }
+  const colsSpanClass = colsClasses[numOfCols]
+
+  return (
+    <div className="bg-slate-500 py-8">
+      <div className="container grid grid-cols-4 lg:grid-cols-12 gap-x-2">
+        {cards &&
```

Comment:
Why not use the ShadCN `<Card>`?

Reply:
Completely forgot about ShadCN.. updated in [c1e5c9c](https://github.com/NWACus/web/pull/141/commits/c1e5c9c117d374335741e2600e68902bda8051fa)